File: PPPK_Web/PPPK_Web/Models/Vozac.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace PPPK_Web.Models
{
    public class Vozac
    {
        public int IDVozac { get; set; }
        public string Ime { get; set; }
        public string Prezime { get; set; }
        public string BrojMobitela { get; set; }
        public string BrojVozacke { get; set; }
        public string PunoIme { get => Ime + " " + Prezime; }

        public Vozac(int iDVozac, string ime, string prezime, string brojMobitela, string brojVozacke)
        {
            IDVozac = iDVozac;
            Ime = ime;
            Prezime = prezime;
            BrojMobitela = brojMobitela;
            BrojVozacke = brojVozacke;
        }

        public Vozac()
        {
        }
    }
}

File: PPPK_Java/src/hr/algebra/model/Vozac.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package hr.algebra.model;

/**
 *
 * @author Marko
 */
public class Vozac {
    
    private int idVozac;
    private String ime;
    private String prezime;
    private String brojMobitela;
    private String brojVozacke;

    public Vozac(String ime, String prezime, String brojMobitela, String brojVozacke) {
        this.ime = ime;
        this.prezime = prezime;
        this.brojMobitela = brojMobitela;
        this.brojVozacke = brojVozacke;
    }

    public Vozac(int iDVozac, String ime, String prezime, String brojMobitela, String brojVozacke) {
        this(ime, prezime, brojMobitela, brojVozacke);
        this.idVozac = iDVozac;
    }

    public int getiDVozac() {
        return idVozac;
    }

    public String getIme() {
        return ime;
    }

    public String getPrezime() {
        return prezime;
    }

    public String getBrojMobitela() {
        return brojMobitela;
    }

    public String getBrojVozacke() {
        return brojVozacke;
    }

    @Override
    public String toString() {
        return String.format("%s %s", ime, prezime);
    }
    
}
